controllers/mdutil/util.go

```diff
@@ -41,6 +42,7 @@ import (
 const (
 	// DefaultMachineDeploymentUniqueLabelKey is the label applied to Machines
 	// in a MachineDeployment containing the hash of the template.
+	// Deprecated: This field package is removed, please use the one in the API package instead.
 	DefaultMachineDeploymentUniqueLabelKey = "machine-template-hash"
```

Comment:
I think we also have to deprecate the other constants

Reply:
They should be all already deprecated by https://github.com/kubernetes-sigs/cluster-api/pull/5241

Reply:
Ups, yeah. Probably looked at the wrong version of the code.

Reply:
renamed the constant as `MachineDeploymentUniqueLabel`